docs/deploy.md

```diff
@@ -0,0 +1,41 @@
+## How to deploy changes to Netlify
+
+### Using Github
+
+The web application is being deployed through Netlify and is linked directly to the Github Repository.
+Changes to data files should be pushed to the Github repository and a new build will be automatically deployed through Netlify.
+
+### Updating
+
```

Comment:
for some people they'll be updating for the first time and they're going to need an indication that they should clone the repo first so just add a brief instruction about that